packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTable/MyAdsTable.scss

```diff
@@ -21,6 +21,11 @@
 
             &__content {
                 height: 100%;
+
+                @include mobile {
+                    // stylelint-disable-next-line declaration-no-important
+                    height: calc(100vh - 21rem) !important;
```

Comment:
is it possible to not use `!important` here?

Reply:
had to use important as height is set to 100% inside table component as inline style.